packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -273,7 +274,7 @@ export class InputManager {
     /** @internal */
     public _pickMove(pointerId: number): PickingInfo {
         const scene = this._scene;
-        const pickResult = scene.pick(
+        const pickResult: PickingInfo = scene.pick(

```

Comment:
nit: not sure why this was added

Reply:
I, for the life of me, can't remember why I did this.  Removed.